blog/2023-01-10-pulsar-2022-year-in-review.md

```diff
@@ -0,0 +1,109 @@
+
+## Project releases
+
+In 2022, the Pulsar community worked hard to improve the project’s capabilities and fix existing bugs with 1 major version and 8 minor versions. Many important Pulsar capabilities were delivered in these releases, such as [cluster-level automatic failure recovery](https://github.com/apache/pulsar/pull/13316) and [lazy-loading added to PartitionedProducer](https://github.com/apache/pulsar/pull/10279). Check out each release below to see details.
+
+- [2.7.5](https://github.com/apache/pulsar/releases/tag/v2.7.5)
+- [2.8.2](https://github.com/apache/pulsar/releases/tag/v2.8.2)
+- [2.8.3](https://github.com/apache/pulsar/releases/tag/v2.8.3)
+- [2.8.4](https://github.com/apache/pulsar/releases/tag/v2.8.4)
+- [2.9.2](https://github.com/apache/pulsar/releases/tag/v2.9.2)
+- [2.9.3](https://github.com/apache/pulsar/releases/tag/v2.9.3)
+- [2.10.0](https://github.com/apache/pulsar/releases/tag/v2.10.0)
+- [2.10.1](https://github.com/apache/pulsar/releases/tag/v2.10.1)
+- [2.10.2](https://github.com/apache/pulsar/releases/tag/v2.10.2)
+
+For more information, see the [Release Notes page](https://pulsar.apache.org/release-notes/).
+
+Updates about clients, Pulsar Manager, and Pulsar Helm Chart are listed below:
+
+- [Pulsar C++ Client 3.0.0](https://github.com/apache/pulsar-client-cpp/releases/tag/v3.0.0)
+- [Pulsar C++ Client 3.1.0](https://github.com/apache/pulsar-client-cpp/releases/tag/v3.1.0)
+- [Pulsar Go Client 0.8.0](https://github.com/apache/pulsar-client-go/releases/tag/v0.8.0)
+- [Pulsar Go Client 0.8.1](https://github.com/apache/pulsar-client-go/releases/tag/v0.8.1)
+- [Pulsar Go Client 0.9.0](https://github.com/apache/pulsar-client-go/releases/tag/v0.9.0)
+- [Pulsar Node.js client 1.6.2](https://github.com/apache/pulsar-client-node/releases/tag/v1.6.2)
+- [Pulsar Node.js client 1.7.0](https://github.com/apache/pulsar-client-node/releases/tag/v1.7.0)
+- [Pulsar Python Client 3.0.0](https://github.com/apache/pulsar-client-python/releases/tag/v3.0.0)
+- [Pulsar Manager 0.3.0](https://github.com/apache/pulsar-manager/releases/tag/v0.3.0)
+- [Pulsar Helm Chart 3.0.0](https://github.com/apache/pulsar-helm-chart/releases/tag/pulsar-3.0.0)
+
+For more information, see the [Clients Release Notes page](https://pulsar.apache.org/release-notes/clients/).
+
+## Pulsar ecosystem
+
+In 2022, the Pulsar community worked with other open-source communities to add more integrations to the Pulsar ecosystem. Notable integrations include:
+
+- [Spring for Apache Pulsar](https://spring.io/blog/2022/09/20/spring-for-apache-pulsar-0-1-0-m1-is-now-available): This project aims to provide Spring-friendly APIs, building blocks, and programming models for writing Java applications that interact with Apache Pulsar. Read this [blog](https://streamnative.io/blog/release/2022-09-21-announcing-spring-for-apache-pulsar/) to see details.
```

Comment:
The StreamNative blog post is outdated. I'd suggest providing a link to the official Spring blog post about 0.1.0 release. /cc @cbornet 
```suggestion
- [Spring for Apache Pulsar](https://spring.io/blog/2022/09/20/spring-for-apache-pulsar-0-1-0-m1-is-now-available): This project aims to provide Spring-friendly APIs, building blocks, and programming models for writing Java applications that interact with Apache Pulsar. Please check the blog post [Spring for Apache Pulsar 0.1.0 available now](https://spring.io/blog/2022/12/15/spring-for-apache-pulsar-0-1-0-available-now) for most latest details and references to documentation.
```

Reply:
Updated. Thanks.